### PR TITLE
add truncation strategy to mysql and postgresql

### PR DESCRIPTION
**Pull request by yads**

### CODE REVIEW

File: lib/database-cleaner.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ var DatabaseCleaner = module.exports = function(type, config) {
       var length = tables.length;
       var tableName = 'Tables_in_' + db.config.database;
       var skippedTables = config.mysql.skipTables;
+      var strategy = config.mysql.strategy || 'deletion';
+      if (strategy !== 'deletion' && strategy !== 'truncation') {
+        return callback('Invalid deletion strategy: ' + strategy);
```
Comment (emerleite) [inline lib/database-cleaner.js]: What about an Error instead of a simple string?

File: lib/database-cleaner.js [hunk 2]
```diff
@@ -79,28 +84,54 @@ var DatabaseCleaner = module.exports = function(type, config) {
       var count  = 0;
       var length = tables.rows.length;
       var skippedTables = config.postgresql.skipTables;
+      var strategy = config.postgresql.strategy || 'deletion';
+      if (strategy !== 'deletion' && strategy !== 'truncation') {
+        return callback('Invalid deletion strategy: ' + strategy);
```
Comment (emerleite) [inline lib/database-cleaner.js]: What about an Error instead of a simple string?

**Comment by emerleite** [comment]
@yads just do the few adjustments and I'll merge it. 

Thanks for the PR ;)

**Comment by yads** [comment]
@emerleite done. Also let me know if you want some help maintaining this project.

**Comment by emerleite** [comment]
@yads version 0.13.0 is on the road.

**Pull request merged**